packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -1470,7 +1468,6 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call
     } else if ([@"dispose" isEqualToString:call.method]) {
       [_registry unregisterTexture:cameraId];
       [_camera close];
-      _dispatchQueue = nil;
```

Comment:
nits:
Now that we don't nullify `_dispatchQueue`, `_dispatchQueue`'s creation can be moved to the init method of the plugin to better indicate its lifecycle.

Reply:
Oops, my page is out of sync. Didn't see Stuart's comment when I was reviewing. I'm going to close this comment as it says the same thing Stuart has said.